includes/Admin/Cross_Origin_Isolation.php

```diff
@@ -135,6 +135,10 @@ protected function replace_in_dom( $html ) {
 		];
 
 		$processed = [];
+
+		// Workaround for hardcoded underscore references to video shortcode.
+		$html = str_replace( 'class="wp-video-shortcode', 'crossorigin="anonymous" class="wp-video-shortcode', $html );
```

Comment:
Let's make sure we address this thoroughly in #7560 👍

Reply:
@swissspidy Should we make https://github.com/google/web-stories-wp/pull/7605 part of 1.7.2 as well?